pipeline-module/{{cookiecutter.handle|lower|slugify}}/.gitignore

```diff
@@ -1,5 +1,5 @@
 # Published by CI
-{{ cookiecutter.handle|lower|slugify }}.pdf
+{{ cookiecutter.handle }}.pdf
```

Comment:
Why no lowercase, etc. any more?

Reply:
The PDF file is the uppercase handle, so that's why the ignore is for the uppercase handle as well. The repo itself is still lowercase.